src/libprojectM/MilkdropPreset/CustomShape.cpp

```diff
@@ -277,54 +277,21 @@ void CustomShape::Draw()
                              static_cast<float>(*m_perFrameContext.border_g),
                              static_cast<float>(*m_perFrameContext.border_b),
                              static_cast<float>(*m_perFrameContext.border_a));
-            glLineWidth(1);
+            if (m_thickOutline)
+            {
+                glLineWidth(2);
```

Comment:
Using glLineWidth >1 is not supported on all platforms/drivers (e.g. GLES) and will have really dirty artifacts on those which support it. Plus it mostly disables anti-aliasing. We should keep it that way until we have proper custom shader-based line rendering in place. See issue #682

Reply:
Removed, this was a local commit not intended for upstream.